### PR TITLE
FOGL-8867 Add config cache size and log level documentation to tuning guide

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

File: docs/tuning_fledge.rst [hunk 1]
```diff
@@ -30,6 +32,42 @@ Many factors will impact the performance of a Fledge system
...
+
+Fledge writes logs via the *syslog* facility of Linux, this allows for multiple different log levels. Altering the log level will impact the performance of the system and can use significant disk space when set to *debug* or *info* levels. Each of the services within a Fledge instance may have the log level set of that service individually.
+
+The logging level for the Fledge core can be set in the *Logging* configuration category in the *Advanced* parent category that can be accessed from the *Configuration* menu item.
+
++------------------+
+| |core_log_level| |
++------------------+
+
+The logging level can be set to *error*, *warning*, *info* or *debug*, the default setting is *warning*. The level set defines the least severe error that will be logged, logs or higher severity that that set will also be logged. In the default setting both *error* and *warning* level logs will be sent to the syslog facility.
```
Comment (ashish-jabble) [inline docs/tuning_fledge.rst:46]: In core - we have one more logging level support i.e 'critical'

At the moment we haven't used this level in python core.

Consistency wise on both sides, We may need to remove this option level from core logging?

Comment (ashish-jabble) [inline docs/tuning_fledge.rst:46]: that that -> that

File: docs/tuning_fledge.rst [hunk 2]
```diff
@@ -30,6 +32,42 @@ Many factors will impact the performance of a Fledge system
...
+
++------------------+
+| |storage_config| |
++------------------+
+
+The south and north services also have log level settings for each service that can be accessed via the *Advanced* tab within the configuration of each of the services.
+
+All other optional services will also have a log level setting accessible via the configuration for each service.
+
+It is recommended to only set the log level of a service to *info* or *debug* when actively tracing a problem with the configuration or operation of a service and to always run with the default, *warning*, log level in production.
```
Comment (ashish-jabble) [inline docs/tuning_fledge.rst:58]: We better can make this as a **note** highlight

File: docs/tuning_fledge.rst (hunk 1 above)
Comment (MarkRiddoch) [reply docs/tuning_fledge.rst:46]: We should probably drop the critical option

**Review by ashish-jabble — APPROVED** [review]

File: docs/tuning_fledge.rst (hunk 1 above)
Comment (ashish-jabble) [reply docs/tuning_fledge.rst]: Sure, let me create a JIRA for it

**Pull request merged**